examples/getRawBlob.js

```diff
@@ -12,7 +10,7 @@ var github = new Client({
 
 github.authenticate({
   type: 'oauth',
-  token: testAuth['token']
+  token: 'add-your-real-token-here'
```

Comment:
for examples like this one where an auth token is not needed, I'd remove the `github.authenticate()` call altogether.

Also note that I changed `owner` and `repor` in all examples, so in this file you need to adapt the sha to point to one in the current repository.